config/task-config.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
   svgSprite   : true,
   ghPages     : true,
   stylesheets : true,
+  icons: true,
```

Comment:
Ah, the icons task is for doing svg sprites, which we can talk about if you're interested, but I don't think you'd need that for this project.

Reply:
Actually, that's not totally right. The svgSprite task is for that. I don't think blendid has an icons task so you could take out this line in the task config.

Reply:
Disregard. I'll move this comment to PR #3.